### ISSUE TITLE
Contrast limit adjustment is tedious if it is far off

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#1020]
One has to open / adjust / close the contrast adjustment click B multiple times in case one needs to change this a lot.

Could we implement some log-scale here?

**Comment by ThomasMicroscopy** [comment]
I agree, this could be improved. Also, what about a auto contrast button. I know this could be difficult, since one first would have to known the histogram of the images, which for a large dataset would take time.
